hydra_redis/redis_usage

```diff
@@ -0,0 +1,19 @@
+Requirements for redis_graph.py file acoording to my local env.
+    install redis-graph ref: http://redisgraph.io/#linux-ubuntu-1604
+    install and load module redis ref: http://redisgraph.io/#loading-the-module-to-redis
+    now for python client usage:
+    pip install redis
+    pip install redisgraph       // you can change the client.py with your requirements
+    and then first run the redis server and then run redis_graph.py file
+    
+    
+    
+After install redis-graph, redis and its python client 
+you should do :
+    $cd redis
+    $src/redis-server --loadmodule /path_to_redis_graph/src/redisgraph.so
+    like  this:
+    src/redis-server --loadmodule /home/sandeep/demo/redis-module-graph/src/redisgraph.so
+ it will run the redis server then you can run your redis file in different terminal but in same env.   
+
+    **docker image will provided early for now you can use the same configuration.
```

Comment:
You should create a bash file or something to setup Redis.

Reply:
I am trying to make a docker image for that. I'll provide it early.